src/mapping/tests/PartitionOfUnityMappingTest.cpp

```diff
@@ -366,9 +366,9 @@ void performTestConsistentMapDeadAxis(Mapping &mapping, int dim)
   double value2 = outData->values()(2);
   BOOST_TEST(mapping.hasComputedMapping() == true);
   if (dim == 2) {
-    BOOST_TEST(value == 19935.268150244759);
-    BOOST_TEST(value1 == 19935.266962237896);
-    BOOST_TEST(value2 == 19935.276990769267);
+    BOOST_TEST(value == 19935.374757794027);
+    BOOST_TEST(value1 == 19935.37795225389);
+    BOOST_TEST(value2 == 19935.35164938603);
```

Comment:
Why was this necessary? Seems that we should use less valid digits here.

Reply:
but then it would be more sustainable to reduce the test accuracy here, right? otherwise, we eventually run into the same problem again.